autoscheduler/user_sessions/tests/name_api_tests.py

```diff
@@ -0,0 +1,64 @@
+from rest_framework.test import APITestCase
+from django.contrib.sessions.models import Session
+from django.contrib.auth.models import User
+
+class UsersFullNameAPITests(APITestCase):
+    """ Tests the functionality of the get_full_name API """
+    def setUp(self):
+        """ Delete sessions table before each test to ensure new session """
+        Session.objects.all().delete()
+
+    def test_get_full_name_returns_correct_value_bob(self):
+        """ Tests that /sessions/get_full_name response is equal to the user's name """
+        #Arrange
+        username = 'user_bob'
+        password = 'password_bob'
+        user = User.objects.create(username=username, first_name='Bob',
+                                   last_name='Bobbins')
+        user.set_password(password)
+        user.save()
+
+        expected_name = 'Bob Bobbins'
+        expected = {'fullName': expected_name}
+
+        #Act
+        self.client.login(username=username, password=password)
+        response = self.client.get(f'/sessions/get_full_name')
```

Comment:
Since these strings don't actually need formatting, use normal strings instead of f-strings